cogboard-app/src/main/kotlin/com/cognifide/cogboard/widget/type/PullRequestReminderWidget.kt

```diff
@@ -0,0 +1,44 @@
+package com.cognifide.cogboard.widget.type
+
+import com.cognifide.cogboard.config.service.BoardsConfigService
+import com.cognifide.cogboard.widget.AsyncWidget
+import io.vertx.core.Vertx
+import io.vertx.core.json.JsonObject
+import com.cognifide.cogboard.CogboardConstants.Props
+class PullRequestReminderWidget(
+    vertx: Vertx,
+    config: JsonObject,
+    serv: BoardsConfigService
+) : AsyncWidget(vertx, config, serv) {
+
+    private val path: String = config.getString("path", "")
+
+    override fun handleResponse(responseBody: JsonObject) {
+        var pullRequests = responseBody
+
+        if (url.contains("github")) {
+            pullRequests = pullRequests
+            .put(Props.PULL_REQUESTS, responseBody.remove("array"))
+            .put(Props.REPOSITORY_HUB, "github")
+        } else {
+            pullRequests = pullRequests
+            .put(Props.PULL_REQUESTS, responseBody.remove("values"))
+            .put(Props.REPOSITORY_HUB, "bitbucket")
+        }
+
+        send(pullRequests)
+    }
+
+    override fun updateState() {
+        if (url.isNotBlank() && path.isNotBlank()) {
+            if (url.contains("github")) {
+                val apiUrl = url.replace("github.com/", "api.github.com")
```

Comment:
This will not work for commertial repos i think you should be replacing `://` with `://api.`

Reply:
my project repo as an example: https://github.company.com/

Reply:
Fixed in commit f03e91e